site/docs/pages/index.mdx

```diff
@@ -461,17 +461,7 @@ bun add @coinbase/onchainkit
   </div>
 </section>
 
-{' '}
-
-<section className="css-alternate-container w-full flex items-center justify-center">
-  <aside className="flex flex-col pb-10 md:flex-row md:px-2 w-[940px]">
```

Comment:
Why removing in it?

What's the long term plan for this?

How do we tell the story around Figma

Reply:
We are removing the image because the current image is meant to be interactive. It's odd to see the zoom in and out buttons for example. 

<img width="208" alt="Screenshot 2024-08-21 at 10 40 43 AM" src="https://github.com/user-attachments/assets/e9993147-da8a-445c-8cd3-9e64b7882c93">

Reply:
So I was thinking that we could update to a more appropriate image that is meant to be static. Or perhaps just remove it until we can implement the interactive version.

Reply:
I agree that this doesn't tell a story. I'd recommend to update to a more flattering image such as:
![image](https://github.com/user-attachments/assets/24649e43-b831-43d8-aba7-fc35d54cf82d)

Reply:
We should update for something more interesting, something like about the Logic of components. That's the fun part and super useful.

<img width="971" alt="Screenshot 2024-08-21 at 11 13 15 AM" src="https://github.com/user-attachments/assets/5745df87-3929-4925-8790-853d3f167f14">

Reply:
Maybe for Swap, just focus on the logic part.
<img width="900" alt="Screenshot 2024-08-21 at 11 15 13 AM" src="https://github.com/user-attachments/assets/f302f36c-7f72-477c-a45a-84ba8366fc4f">

Reply:
Those images are zoomed so far out you can't read anything

Reply:
Adam was talking about how embedding the interactive Figma files is easy. So maybe we just wait for that update

Reply:
Ok, but what's the story telling we want here? I don't think embedding is the answear. We want to tease the fact we have an OSS Figma, we don't want to distract people.

So howe can we tease without distracting?

@cpcramer again, the question always is, "what's the story we are telling, and how we can be intentional about it."

If we don't know the story, than not even the person reading will know that.